packages/babel-helper-function-name/src/index.ts

```diff
@@ -223,6 +225,10 @@ export default function (
     name = id.name;
   }
 
+  if (!supportUnicodeId && isFunction(node) && /[\uD800-\uDFFF]/.test(name)) {
+    return;
+  }
```

Comment:
nit: This test is more expensive, consider place it after the `if (name === undefined)` check.

Reply:
Okay, I will move it down.

Reply:
Did you commit the changes?